services/movie.py

```diff
@@ -0,0 +1,32 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: int = None, actors_ids: int = None) -> str:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids, actors__id__in=actors_ids)
+
+    if genres_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        return Movie.objects.filter(actors__id__in=actors_ids)
+
+
+def get_movie_by_id(movie_id: int) -> str:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(movie_title: str,
+                 movie_description: str,
+                 genres_ids: list = None,
```

Comment:
Using a mutable default argument like `list` in function definitions can lead to unexpected behavior. Consider using `None` as the default value and then initializing the list inside the function if needed.